docs/docs/cmd/pa/app/app-permission-list.mdx

```diff
@@ -0,0 +1,124 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# pa app permission list
+
+Lists all permissions of a Power Apps app
+
+## Usage
+
+```sh
+m365 pa app permission list [options]
+```
+
+## Options
+
+```md definition-list
+`---appName <appName>`
+: The name (GUID) of the Microsoft Power App.
+
+`--asAdmin`
+: Run the command as admin for apps you don't own.
+
+`-e, --environmentName [environmentName]`
+: The name of the environment. Required when using `asAdmin`.
+
+`--roleName [roleName]`
```

Comment:
Let's put this option behind `appName` this way the non-admin and admin options are kinda grouped together.